src/Illuminate/Broadcasting/Broadcasters/Ably/Utils.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Illuminate\Broadcasting\Broadcasters\Ably;
+
+/**
+ * @author Sachin Shinde (sachinshinde7676@gmail.com)
+ */
+class Utils
+{
+    // JWT related PHP utility functions
+    /**
+     * @param  string  $jwt
+     * @return array
+     */
```

Comment:
Please make sure all docblocks are in the same format as the rest of the framework. Each one needs a small one line explanation on top.